version

```diff
@@ -1 +1 @@
-1.3.43
+1.3.44
```

Comment:
it is better to keep the version here match with the repo tag. The latest tag is v1.3.49, so this version should be updated to 1.3.50 ( the next tag that's going to be created).

Reply:
I just updated the [wiki instructions](https://github.com/hpe-hcss/quake/wiki/Updating-Project-API) to note to use the same version as the next release ta.